test/extensions/common/wasm/test_data/test_context_cpp.cc

```diff
@@ -0,0 +1,46 @@
+// NOLINT(namespace-envoy)
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#ifndef NULL_PLUGIN
+#include "proxy_wasm_intrinsics.h"
+#include "source/extensions/common/wasm/ext/envoy_proxy_wasm_api.h"
```

Comment:
Can't we separate between this and wasm binary? I think that these proxy-specific extensions should be separated from envoy implementation like proxy-wasm. Do you have any idea to overcome this?

Reply:
Do you mean we should create an alias in bazel so that these appear on a different directory such that we could create an Envoy specific SDK repo with the same header locations?  We could do that in a follow on PR.

Reply:
It means that whether we can define these headers as single github repository. So to speak, whether we can create a repository like `envoyproxy/envoy-proxy-wasm-cpp-sdk`. This header implementation seems to be completely separated from upstream envoy so that we can achieve this. It will be more understandable for filter developers.

Reply:
We should do that once we snap a stable LTS version 1 ABI/SDK. Let me open an issue in envoyproxy/envoy: https://github.com/envoyproxy/envoy/issues/11755

Reply:
We will need a new repo under envoyproxy so we need Matt to create that.